docs/virtual-machine-actions.md

```diff
@@ -0,0 +1,59 @@
+# Virtual Machine Actions
+
+## Listing
+Lists all of the current virtual machines. You can do this with `vms list`. Takes either `--id` or a subdomain for the organization:
+
+```
+$ katapult vms list debug-inc
++-------+--------------------------------+
+| NAME  |              FQDN              |
++-------+--------------------------------+
+| demo  | testing.debug-inc.st.kpult.com |
+| hello | world.debug-inc.st.kpult.com   |
++-------+--------------------------------+
+```
+
+## Power Actions
+There are various power actions you can perform with VM's:
+- `vms poweroff <--fqdn or --id>`: Powers off a virtual machine.
```

Comment:
Maybe provide an example with --fqdn and --id, and then just say the parameters are the same for all the power commands. The `<--fqdn or --id>` is a bit confusing at first.